webpack.config.js

```diff
@@ -34,15 +35,43 @@ const includeFeature = registry ? (extension, id) => {
   return include.every(test) && !exclude.some(test);
 } : () => true;
 
+const tsLoader = [
+  {
+    loader: 'awesome-typescript-loader'
+  }
+];
+
+const tsLoaderDev = [
+  {loader: 'cache-loader'},
+  {
+    loader: 'thread-loader',
+    options: {
+      // there should be 1 cpu for the fork-ts-checker-webpack-plugin
+      workers: require('os').cpus().length - 1
+    }
+  },
+  {
+    loader: 'ts-loader',
+    options: {
+      happyPackMode: true, // IMPORTANT! use happyPackMode mode to speed-up compilation and reduce errors reported to webpack,
```

Comment:
The new TypeScript compilation is so much faster! Can we apply this to all Caleydo apps/projects by simply copying the new webpack.config.js and the updates from the .yo-rc file? Does `yo phovea:update` apply this change already?